samples/ldo/src/main.c

```diff
@@ -22,13 +22,15 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 static void get_and_log_voltage(npmx_ldsw_t *p_ldsw)
 {
 	npmx_ldsw_voltage_t ldsw_voltage;
+	uint32_t voltage_mv;
 
 	if (npmx_ldsw_ldo_voltage_get(p_ldsw, &ldsw_voltage) != NPMX_SUCCESS) {
 		LOG_ERR("Unable to get voltage status.");
 	}
 
-	LOG_INF("Voltage LDO%d: %d mV.", (p_ldsw->hw_index + 1),
-		npmx_ldsw_voltage_convert_to_mv(ldsw_voltage));
+	NPMX_ASSERT(npmx_ldsw_voltage_convert_to_mv(ldsw_voltage, &voltage_mv) == true);
```

Comment:
I think assertion is not needed here, it is already checked if  `npmx_ldsw_ldo_voltage_get` succeeded. And if value returned by `npmx_ldsw_voltage_convert_to_mv` needs to be checked, then I think assertion is too strong and simple if with `LOG_ERR` would be enough.